### ISSUE TITLE
UnicodeDecodeError in Python 2.7 prevents debugging

### ISSUE THREAD

**Issue by karthiknadig** [Microsoft/ptvsd#845]
## Environment data

- PTVSD version: master
- OS and version: windows
- Python version (& distribution if applicable, e.g. Anaconda): 2.7
- Using VS Code or Visual Studio: VS

## Actual behavior

Debugging fails with this error:
```
ERROR: 'setDebuggerProperty' request failed!

ProtocolException: Traceback (most recent call last):
File "C:\Python27\lib\site-packages\ptvsd\ipcjson.py", line 358, in on_request
target(request, args)
File "C:\Python27\lib\site-packages\ptvsd\wrapper.py", line 1271, in f
fut = m(self, self.loop, *args, **kwargs)
File "C:\Python27\lib\site-packages\ptvsd\futures.py", line 140, in g
it = f(self, *args, **kwargs)
File "C:\Python27\lib\site-packages\ptvsd\wrapper.py", line 2325, in on_setDebuggerProperty
self._apply_code_stepping_settings()
File "C:\Python27\lib\site-packages\ptvsd\wrapper.py", line 1421, in _apply_code_stepping_settings
'\t'.join(project_dirs))
File "C:\Python27\lib\site-packages\ptvsd\wrapper.py", line 1300, in pydevd_request
return self._pydevd_request(self.loop, cmd_id, args)
File "C:\Python27\lib\site-packages\ptvsd\wrapper.py", line 436, in pydevd_request
seq, s = self.make_packet(cmd_id, args)
File "C:\Python27\lib\site-packages\ptvsd\wrapper.py", line 421, in make_packet
s = u'{}\t{}\t{}\n'.format(cmd_id, seq, args)
UnicodeDecodeError: 'ascii' codec can't decode byte 0xce in position 3: ordinal not in range(128)
```

## Expected  behavior

Should not fail with `UnicodeDecodeError`

Bug report: https://developercommunity.visualstudio.com/content/problem/332784/vs-python-%E9%A1%B9%E7%9B%AE%E5%88%87%E6%8D%A2%E7%89%88%E6%9C%AC%E6%97%A0%E6%B3%95%E8%BF%90%E8%A1%8C.html

**Comment by abhinuvpitale** [comment]
The input should be checked and there should be exception handling here, I can take this up!

**Comment by karthiknadig** [comment]
Fixed via #884 . this code path is no longer needed.
